### PR TITLE
fix(portal): fix issue with api path display

### PR DESCRIPTION
**Pull request by james-tran-3005**
## Description

- Make all api path display left truncated when there is no space to accommodate the full path. Hover on truncated path will show a tooltip with full path displayed
- Fix seller api description default shows 'description' despite of holding no value
- Fix api spec table method overlaps with api path

## Related Issue

- [[1.0][rc]API path display improvement](https://github.com/mycloudnexus/kraken/issues/162)
- [[BUG] [FE] seller api setup - view api - overlap issue on 'Method' area](https://github.com/mycloudnexus/kraken/issues/182)
- [[1.0][rc][BUG] [FE] seller api set up page - description has default value](https://github.com/mycloudnexus/kraken/issues/183)

## Screenshots (if appropriate)
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] CI/CD or documentation update (changes to CI/CD pipeline or documentation)

## Checklist

- [ ] I ensured that the documentation is up to date
- [x] I explained why this PR updates in detail with reasoning why it's required
- [x] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Review by KsiBart — DISMISSED** [review]

File: kraken-app/kraken-app-portal/src/__test__/pages/Deployment/DeploymentHistory.test.tsx [hunk 1]
```diff
@@ -34,4 +34,51 @@ describe("Deployment > Deployment history tests", () => {
...
+            "mappingMatrix": {
+              "productType": "access_e_line",
+              "actionType": "delete"
+            },
+            "componentName": "Product Ordering Management",
+            "componentKey": "mef.sonata.api.order",
+            "envId": "7da9098b-107f-402c-9d6d-20b1be96b99a",
+            "envName": "production",
+            "createAt": "2024-11-18T10:06:14.412978Z",
+            "createBy": "24044233-3683-425e-9cbe-62e643f081d8",
```
Comment (zhiyaoPCCW) [inline kraken-app/kraken-app-portal/src/__test__/pages/Deployment/DeploymentHistory.test.tsx:59]: please change the testing data, do not expose personal name.

**Review by zhiyaoPCCW — CHANGES_REQUESTED** [review]

File: kraken-app/kraken-app-portal/src/__test__/pages/Deployment/DeploymentHistory.test.tsx (hunk 1 above)
Comment (zhiyaoPCCW) [reply kraken-app/kraken-app-portal/src/__test__/pages/Deployment/DeploymentHistory.test.tsx:59]: Also please make sure there is no code smell in pr.

Comment (james-tran-3005) [reply kraken-app/kraken-app-portal/src/__test__/pages/Deployment/DeploymentHistory.test.tsx:58]: I copied api response and forgot to change it. Won't producing same issue again.

**Review by zhiyaoPCCW — APPROVED** [review]
LGTM

**Pull request merged**